src/components/NotificationPermissionPrompt.tsx

```
@@ -0,0 +1,116 @@
+      <Card className={`border-green-200 bg-green-50 ${className || ''}`}>
+        <CardContent className="pt-6">
+          <div className="flex items-center space-x-3">
+            <div className="flex-shrink-0">
+              <Check className="h-5 w-5 text-green-600" />
+            </div>
+            <div className="flex-1">
+              <p className="text-sm font-medium text-green-800">
+                Notifications enabled successfully!
+              </p>
+              <p className="text-xs text-green-600 mt-1">
+                You'll now receive alerts when your Pomodoro timers complete.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)